src/plugins/console/server/services/spec_definitions_service.ts

```diff
@@ -115,8 +116,8 @@ export class SpecDefinitionsService {
   }
 
   private loadJSONSpecInDir(dirname: string) {
-    const generatedFiles = globby.sync(join(dirname, 'generated', '*.json'));
-    const overrideFiles = globby.sync(join(dirname, 'overrides', '*.json'));
+    const generatedFiles = globby.sync(normalizePath(join(dirname, 'generated', '*.json')));
```

Comment:
nit: should we leave a comment here explainig why we are normalizing the path to avoid problems in the future?